INN/umbrella-sfpublicpress
# Add date and byline styling for posts in search results

## Changes

This pull request makes the following changes:

- Updates the `content-search` partial to include the site-wide post date format and byline

Before:
![Screen Shot 2020-05-28 at 12 17 13 PM](https://user-images.githubusercontent.com/18353636/83166617-2f4ccb80-a0dd-11ea-8d9f-f50e58b5832e.png)

After:
![Screen Shot 2020-05-28 at 12 17 01 PM](https://user-images.githubusercontent.com/18353636/83166607-2e1b9e80-a0dd-11ea-845f-234e28364f14.png)

## Why



For #74

## Testing/Questions

Features that this PR affects:

- Search page post results


Questions that need to be answered before merging:


Steps to test this PR:

1. Complete a search via the search bar and then view the results. Make sure the date and byline format match everything else
